src/persist/src/indexed/runtime.rs

```diff
@@ -316,15 +400,17 @@ mod tests {
             (("key2".to_string(), "val2".to_string()), 1, 1),
         ];
 
-        let indexed = Indexed::new(MemBuffer::new("concurrent")?, MemBlob::new("concurrent")?)?;
-        let client1 = start(indexed);
-        let id = client1.register("0")?;
+        let buffer = MemBuffer::new("concurrent")?;
+        let blob = MemBlob::new("concurrent")?;
+        let client1 = start(buffer, blob)?;
+        let _ = client1.register("0")?;
 
         // Everything is still running after client1 is dropped.
         let mut client2 = client1.clone();
         drop(client1);
-        block_on(|res| client2.write(id, &data, res))?;
```

Comment:
I'm not sure how critical it is to test the lower level `write` api instead of `write_sync` -- it seemed ok to me to replace that here but am happy to add it back

Reply:
This works for me. The way I think about this is we should have most tests of write, etc be directly on Indexed and then let these ones focus on stuff like concurrency (which is what Runtime adds on top on Indexed)